src/test/java/racingCar/CarTest.java

```diff
@@ -0,0 +1,40 @@
+package racingCar;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarTest {
+
+    private Car car;
+
+    @BeforeEach
+    void setUp() {
+        car = new Car();
+    }
+
+    @Test
+    void 자동차_처음_위치_0() {
+        assertThat(car.getDistance()).isEqualTo(0);
+    }
+
+
+    @ParameterizedTest
+    @ValueSource(ints = {4, 5, 6, 7, 8, 9})
```

Comment:
```suggestion
    @ValueSource(ints = {4, 9})
```
테스트 데이터가 무조건 많다고 좋은 것은 아니지 않을까?
이렇게 데이터가 쌓이면 전체 테스트 실행 속도도 떨어지고 굳이 불필요한 테스트 데이트는 테스트하지 않는 습관을 들이면 어떨까?
이 경우 경계값만 테스트해도 되지 않을까?